lib/App/Rakubrew/Shell/Zsh.pm

```diff
@@ -110,10 +110,9 @@ sub get_shell_unsetter_code {
 
 sub completions {
     my $self = shift;
-    my $index = shift;
```

Comment:
zsh auto-completion works fine here. The zsh function `_${brew_name}_completions()` (line 83) *does* pass a position argument first. I suspect this change would break things.

Reply:
Right, overlooked the `$POS`. Will revert this.